cluster/common.sh

```diff
@@ -277,7 +282,7 @@ function load-or-gen-kube-basicauth() {
 #   KUBE_VERSION
 function set_binary_version() {
   if [[ "${1}" =~ "/" ]]; then
-    IFS='/' read -a path <<< "${1}"
+    IFS='/' read -a -r path <<< "${1}"
```

Comment:
`read -a -r` can not work. I think `read -r -a` is correct.

Reply:
Thanks!